app/internal/wm/os_js.go

```diff
@@ -583,6 +590,19 @@ func (w *window) windowMode(mode WindowMode) {
 	}
 }
 
+func (w *window) navigationColor(c color.NRGBA) {
+	cssColor := fmt.Sprintf("#%06X", []byte{c.R, c.G, c.B})
```

Comment:
What happened to the alpha?

Reply:
Google Chrome (which is the only one that supports `theme-color`) doesn't supports `alpha`. The best soution is to pre-compute the alpha, but my last attempt doesn't work. 

Maybe something like:

```
r, g, b, _ := color.RGBAModel.Convert(c).RGBA()
cssColor := fmt.Sprintf("#%06X", []uint8{uint8(r/255), uint8(g/255), uint8(b/255)})
```

Or...

```
r, g, b, a := uint32(c.R), uint32(c.G), uint32(c.B), uint32(c.A)
c.R = uint8(a * r / 255)
c.G = uint8(a * g / 255)
c.B = uint8(a * b / 255)
cssColor :=  fmt.Sprintf("#%06X", []byte{c.R, c.G, c.B})
```

Reply:
Perhaps take a look at package internal/f32color.

Reply:
Specifically, use f32color.LinearFromSRGB to convert the color to linear float32-based color channels. Then you can multiply each color channel with alpha. Finally, set alpha to 1.0 and convert back to sRGB with the SRGB method.

Reply:
I take a look at `f32color` and it contains `NRGBAToRGBA_PostAlpha` (https://github.com/gioui/gio/blob/4f9326c6d17c4ec1f5eb86afc0d50f6565c9b688/internal/f32color/rgba.go#L96-L112), can I use that directly? 🤔